ml-conversational-analytic-tool/featureVector.py

```diff
@@ -181,9 +182,11 @@ def pullStringConversation(self, export_filename="", export=True):
                                                                           comment_row["Body"])
             string_conversations.append(conversation.encode("ascii", "ignore").decode())
             pull_urls.append(row["URL"])
+            pull_numbers.append(row["Number"])
 
         # Export converation field dataset
```

Comment:
Typo: conversation

Reply:
good catch, looks like there's a lot of the same typos in multiple places. I will leave that out of this PR and make a separate PR for clean up